Flaky test PodTest.testExecExplicitDefaultContainerMissing

```
Error:  Tests run: 32, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 15.68 s <<< FAILURE! -- in io.fabric8.kubernetes.client.mock.PodTest
Error:  io.fabric8.kubernetes.client.mock.PodTest.testExecExplicitDefaultContainerMissing -- Time elapsed: 10.11 s <<< ERROR!
io.fabric8.kubernetes.client.KubernetesClientException: An error has occurred.
	at io.fabric8.kubernetes.client.KubernetesClientException.launderThrowable(KubernetesClientException.java:109)
	at io.fabric8.kubernetes.client.KubernetesClientException.launderThrowable(KubernetesClientException.java:102)
	at io.fabric8.kubernetes.client.utils.Utils.waitUntilReadyOrFail(Utils.java:187)
	at io.fabric8.kubernetes.client.dsl.internal.core.v1.PodOperationsImpl.setupConnectionToPod(PodOperationsImpl.java:387)
	at io.fabric8.kubernetes.client.dsl.internal.core.v1.PodOperationsImpl.exec(PodOperationsImpl.java:293)
	at io.fabric8.kubernetes.client.mock.PodTest.testExecExplicitDefaultContainerMissing(PodTest.java:591)
	at java.base/java.lang.reflect.Method.invoke(Method.java:569)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
Caused by: java.util.concurrent.TimeoutException: not ready after 10000 MILLISECONDS
```

--- a/kubernetes-tests/src/test/java/io/fabric8/kubernetes/client/mock/PodTest.java
+++ b/kubernetes-tests/src/test/java/io/fabric8/kubernetes/client/mock/PodTest.java
@@ -562,11 +562,11 @@ class PodTest {
   }
 
   @Test
-  void testExecExplicitDefaultContainerMissing() {
+  void testExecExplicitDefaultContainerMissing() throws InterruptedException {
     server.expect()
         .withPath("/api/v1/namespaces/test/pods/pod1/exec?command=ls&container=first&stderr=true")
         .andUpgradeToWebSocket()
-        .open()
+        .open(new ErrorStreamMessage("err"))
         .done()
         .always();
 
@@ -585,11 +585,14 @@ class PodTest {
         .once();
 
     // When
+    final CountDownLatch execLatch = new CountDownLatch(1);
     ExecWatch watch = client.pods()
         .withName("pod1")
         .terminateOnError()
+        .usingListener(createCountDownLatchListener(execLatch))
         .exec("ls");
 
+    assertTrue(execLatch.await(1, TimeUnit.MINUTES));
     watch.close();
   }
 

